pkg/package/modfile_test.go

```diff
@@ -235,3 +235,16 @@ func TestGetFilePath(t *testing.T) {
 	assert.Equal(t, mfile.GetModFilePath(), filepath.Join(testPath, MOD_FILE))
 	assert.Equal(t, mfile.GetModLockFilePath(), filepath.Join(testPath, MOD_LOCK_FILE))
 }
+
+func TestGenSource(t *testing.T) {
```

Comment:
TestAddWithGitCommit failed on windows in CI.

Reply:
Fixed it!